Update _mcp-credit-usage.md (#9591)
fishtown-analytics/docs.getdbt.com: website/snippets/_mcp-credit-usage.md

:::info

> Only [`text_to_sql`](/docs/dbt-ai/mcp-available-tools) consumes dbt Copilot actions allotment. Other MCP tools do not.

Reviewer: ```suggestion
Only [`text_to_sql`](/docs/dbt-ai/mcp-available-tools) consumes your dbt Copilot action allotment. Other MCP tools do not.
```

> Only [`text_to_sql`](/docs/dbt-ai/mcp-available-tools) consumes dbt Copilot credits. Other MCP tools do not consume credits.

Reviewer: Hiya Mirna

In Billing, AI usage is described as dbt Copilot actions and a usage limit, not “credits”: “dbt AI usage is measured based on the number of completed AI requests, known as dbt Copilot actions.”

This snippet uses “dbt Copilot credits” instead. I noticed the engs mentioned "credits" in the slack thread but maybe worth double checking in case we want to align with billing language?

> Only [`text_to_sql`](/docs/dbt-ai/mcp-available-tools) consumes dbt Copilot credits. Other MCP tools do not consume credits.

Author: ```suggestion
Only [`text_to_sql`](/docs/dbt-ai/mcp-available-tools) consumes dbt Copilot actions allotment. Other MCP tools do not consume the included action allotment.
```

> Only [`text_to_sql`](/docs/dbt-ai/mcp-available-tools) consumes dbt Copilot actions allotment. Other MCP tools do not consume the included action allotment.

Author: ```suggestion
Only [`text_to_sql`](/docs/dbt-ai/mcp-available-tools) consumes dbt Copilot actions allotment. Other MCP tools do not.
```

> When your account runs out of dbt Copilot actions, the remote MCP server blocks every tool that runs through it, including tools invoked from a local MCP server and [proxied](https://github.com/dbt-labs/dbt-mcp/blob/main/src/dbt_mcp/tools/toolsets.py#L24) to remote MCP, such as SQL and remote Fusion tools.

Reviewer: ```suggestion
When your account runs out of dbt Copilot actions, the remote MCP server blocks every tool that runs through it, including tools invoked from a local MCP server and [proxied](https://github.com/dbt-labs/dbt-mcp/blob/main/src/dbt_mcp/tools/toolsets.py#L24) to remote MCP, such as SQL and remote <Constant name="fusion" /> tools.
```

> However, all remote MCP tools are gated by the Copilot credit limit. When your account has no available dbt Copilot credits, the remote MCP server blocks every tool that runs through it, including tools invoked from a local MCP server and [proxied](https://github.com/dbt-labs/dbt-mcp/blob/main/src/dbt_mcp/tools/toolsets.py#L24) to remote MCP, such as SQL and remote Fusion tools.

Reviewer: Hiya Mirna

This section uses credit limit / credits, while paragraph 3 uses usage limit. Maybe worth aligning if one term is agreed upon.

> However, all remote MCP tools are gated by the Copilot credit limit. When your account has no available dbt Copilot credits, the remote MCP server blocks every tool that runs through it, including tools invoked from a local MCP server and [proxied](https://github.com/dbt-labs/dbt-mcp/blob/main/src/dbt_mcp/tools/toolsets.py#L24) to remote MCP, such as SQL and remote Fusion tools.

Author: ```suggestion
When your account runs out of dbt Copilot actions, the remote MCP server blocks every tool that runs through it, including tools invoked from a local MCP server and [proxied](https://github.com/dbt-labs/dbt-mcp/blob/main/src/dbt_mcp/tools/toolsets.py#L24) to remote MCP, such as SQL and remote Fusion tools.
```

> If you reach your dbt Copilot usage limit, remote MCP tools remain unavailable until the credits reset. If you need help, contact your account manager.

Reviewer: Hiya Mirna

This sentence uses usage limit and credits reset in the same line. If we align on Copilot actions in paragraph 1, should this say something like "until your usage resets" (or "until the start of your next billing cycle") for consistency with [Billing](https://docs.getdbt.com/docs/platform/billing)?

> If you reach your dbt Copilot usage limit, remote MCP tools remain unavailable until the credits reset. If you need help, contact your account manager.

Author: ```suggestion
If you reach your dbt Copilot actions limit, remote MCP tools remain unavailable until the limit resets. If you need help, contact your account manager.
```
